surfer/viz.py

```diff
@@ -1186,7 +1186,7 @@ def add_annotation(self, annot, borders=True, alpha=1, hemi=None,
             hemisphere, i.e. ``annot=(labels, ctab)`` for a single hemisphere
             or ``annot=((lh_labels, lh_ctab), (rh_labels, rh_ctab))`` for both
             hemispheres. ``labels`` and ``ctab`` should be arrays as returned
-            by :func:`nibabel.freesurfer.read_annot`.
+            by :func:`nibabel.freesurfer.io.read_annot`.
```

Comment:
Is nibabel changing its imports? `nib.freesurfer.read_annot` has always worked for me.

Reply:
It is probably imported both places, but this is where it lives in their docs (sphinx index) so this is necessary for it to link properly.

Reply:
Ah got it.